### ISSUE TITLE
Add version identifier to the webpage

### ISSUE THREAD

**Issue by stdedos** [microsoft/playwright.dev#1060]
Like:
![image](https://github.com/microsoft/playwright.dev/assets/133706/b88cba4b-6134-446b-a613-fdfdea574567)

Even better would be a version switcher - but as a first step, _at least_ what-am-I-reading would be nice

**Comment by mxschmitt** [comment]
We used to have a version picker, but then decided we switched to the "since approach".

Closing since we don't plan to add the version picker back! Since we [recommend keeping Playwright up2date](https://playwright.dev/docs/best-practices#keep-your-playwright-dependency-up-to-date) to test on the latest browsers.

**Comment by stdedos** [comment]
_Since you've added that_ (pun intended), can you please explain to me _since when_ is `stdout` / `stderr` arguments? (pun not intended).
_`since`_ might be a nice approach, but it also needs to be done right.

Using the aphorism

> We used to have a version picker, but then decided we switched to the "since approach".
>
> Closing since we don't plan to add the version picker back! Since we [recommend keeping Playwright up2date](https://playwright.dev/docs/best-practices#keep-your-playwright-dependency-up-to-date) to test on the latest browsers.

tells me nothing. _I might think I am on the latest version_. I won't go through my dependency list every single time _something_ is lacking from your documentation.

Also, _Explicit >> Implicit_. I don't understand "what a disaster would be" to keep the version of your library somewhere visible.

Finally, _don't close tickets as completed when you have not implemented them_! Gitlab has two kinds of _Closed_; use them!

**Comment by mxschmitt** [comment]
I apologize, options of plugins, like in your case the stdout/stderr property of the WebServer are indeed not documented "since wise", we can find a way of adding it to the plugins section.

A bit of history so you understand our decision making regarding the version picker:

Originally when we had the old versions in place, it increased the build time significantly, since we have 4 language bindings, it was already taking long time to build and in the end with like 5 versions, it took 40-50 minutes even on a powerful machine to build our docs (for local development as well as well when compiling for production). This was not acceptable for us.

For us this ended up of only having the last 3 versions in the language picker, where we thought, does it bring any value? Seems more like confuse people, and is against our recommendation to stay up-to-date.

> Finally, don't close tickets as completed when you have not implemented them! Gitlab has two kinds of Closed; use them!

This is unfortunately a default by GitHub, which confuses a lot of people. I think we should reach out to them, since there is only won't fix or completed. Happened multiple times already that just pressing close ends up in confusion.

**Comment by stdedos** [comment]
tl;dr: Please reconsider adding at least a version identifier somewhere on the page, as "always on latest" is IMHO a very Utopian perspective. It adds value (albeit small), with negligible-to-no cost. _Long response follows, apologies_.

-----

> Originally when we had the old versions in place, it increased the build time significantly, since we have 4 language bindings, it was already taking long time to build and in the end with like 5 versions, it took 40-50 minutes even on a powerful machine to build our docs (for local development as well as well when compiling for production). This was not acceptable for us.

I understand that. "That's not acceptable" for me either. (Ofc, I'm most importantly thinking: Why do older versions need to be compiled on local dev? Why aren't previous versions compiled separately to the main branch, and just glued together on the remote? - but that's none of my real concern I guess)

> For us this ended up of only having the last 3 versions in the language picker, where we thought, does it bring any value? Seems more like confuse people, and is against our recommendation to stay up-to-date.

If you don't want to go with multi-version documentation, that is fine.
However, my original/first idea was "Add version identifier to the webpage":
i.e., same as you are writing"Copyright © 2023 Microsoft", I'd expecting that I'd also see "this documentation reflects x.y.z version status".

If you don't want to implement both "steps" of the idea - the first one is fine. It might be a reasonable assumption from your point of view; from mine, is "as long as always-updating on the latest version bears risks", I might need to do it at a slower speed instead.

-----

> > Finally, don't close tickets as completed when you have not implemented them! Gitlab has two kinds of Closed; use them!
> 
> This is unfortunately a default by GitHub, which confuses a lot of people. I think we should reach out to them, since there is only won't fix or completed. Happened multiple times already that just pressing close ends up in confusion.

It is. And I am sad that Github does not have both of the buttons available, and instead decides to hide them in a drop down (hot take: There is UI space; just put those buttons side-by-side)

However, please note that the "closing states" are not described as "tersely":
![image](https://github.com/microsoft/playwright.dev/assets/133706/4dd117dc-52b7-40e3-a146-6276c137104c)

If I am reading correctly what you've told me above, I'd say you are _effectively_ saying "won't fix" 😕 (not-a-bug, intended behavior ...).
I get it from your point you are saying "issue addressed", _i.e. I responded and probably there is nothing more to do here_, but on the idea "Add version identifier to the webpage" you are saying "won't consider/do it".

If I am not mistaken, it is still possible to close this _now_ as won't-fix, if that remains the repository's take on the issue:
![image](https://github.com/microsoft/playwright.dev/assets/133706/a21f3b07-81d0-4f64-8c63-7acc73659d5a)
